Handling_Resources.php

```diff
@@ -0,0 +1,74 @@
+class Handling_Resources
```

Comment:
Please Follow PHP SPR-4 name convention for class names, method name,variables names
https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-1-basic-coding-standard.md
https://www.php-fig.org/psr/psr-2/